src/common/ReactHooks.ts

```diff
@@ -14,9 +14,8 @@ export function getRootVNode(): ReactExtended.ReactVNode | undefined {
 	const element = document.querySelector(REACT_ROOT_SELECTOR);
 	if (!element) return undefined;
 
-	const root = Reflect.get(element, "_reactRootContainer");
-
-	return root?._internalRoot?.current;
+	const root = Reflect.get(element, "_reactRootContainer") || Reflect.get(element, "__reactContainer$");
+	return root;
```

Comment:
This changes the return is this correct?

Reply:
for react 16 we returned `root._internalRoot.current` vs now we always just return `root`.

Reply:
Yes, idk why but this is correct